master/introduction/containers/network-architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Network Architecture
```

Comment:
We've already got a reference document which is intended to cover this: http://docs.projectcalico.org/v2.2/reference/architecture/data-path

I think we should update that existing document (it's pretty old and not very friendly) rather than add this here.